packages/components/forms/stories/TextInput.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import { TextInput, TextInputProps } from '../src';
+import { SectionHeading } from '@contentful/f36-typography';
+import { Flex } from '@contentful/f36-core';
+import { SearchIcon } from '@contentful/f36-icons';
+
+export default {
+  title: 'Form Elements/TextInput',
+  component: TextInput,
+  args: {
+    onBlur: undefined,
+    onChange: undefined,
+    onFocus: undefined,
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+};
+
+export const Basic = (args: TextInputProps) => {
+  return <TextInput {...args} name="Example name" id="input-1" />;
+};
+
+Basic.args = {
+  label: 'Example label for input',
+  placeholder: 'this is my placeholder',
+};
+
+export const overview = () => (
```

Comment:
```suggestion
export const Overview = () => (
```

just for code consistency... not really important